### PR TITLE
Adds the ability to delete a workshop

### PR DESCRIPTION
**Pull request by boddhisattva**
Changes as part of this PR address [this issue](https://github.com/codebar/planner/issues/371).

Based on the discussion [here](https://github.com/codebar/planner/issues/371#issuecomment-169356782) I've assumed that a workshop is eligible for deletion if the current time is within 4 hours from the time a workshop is created.

### CODE REVIEW

File: app/controllers/admin/workshops_controller.rb [hunk 1]
```diff
@@ -77,6 +79,19 @@ def invite
...
   end
 
+  def destroy
+    if workshop_has_no_invitees? && workshop_created_within_specific_time_frame?
+      @workshop.destroy
+
+      redirect_to admin_root_path, notice: "Workshop deleted succesfully"
+    else
+      redirect_to admin_workshop_path(@workshop),
+                  notice: "Workshop cannot be deleted. This is because it has invitees" \
```
Comment (despo) [inline app/controllers/admin/workshops_controller.rb]: Can we extract the text to the locale and use i18n to retrieve it.

**Comment by despo** [comment]
@boddhisattva looks great. It would be great if we had the text extracted to the locale file as well.
The configuration looks good!

Happy to merge as is as well as we could always pick up the small changes afterwards. Just let me know!

**Comment by boddhisattva** [comment]
@despo Good point :+1: . I have extracted it out into the locale file. Thanks for your feedback on the PR :)

**Comment by despo** [comment]
@boddhisattva thank you :+1:

**Pull request merged**